wasm/logic/numeric.py

```diff
@@ -710,7 +710,8 @@ def fceil_op(config: Configuration) -> None:
     logger.debug("%s(%s)", instruction.opcode.text, value)
 
     if numpy.isnan(value):
-        config.push_operand(value)
+        with allow_invalid():
+            config.push_operand(numpy.ceil(value))
```

Comment:
Huh, interesting, does `ceil` actually do something here? Or is this equivalent to `push_operand(F64_CANON_NAN)`?

Just not sure what allowing invalid means in this context. If it's a no-op then it seems like it's not worth even running the `numpy.ceil()` function.

Reply:
The `invalid` comes from feeding a `NaN` into `numpy.ceil()`.  Internally, I don't know what the call does, but the tests fail without it.